error running "make instance-setup" step

Getting an error during "make instance-setup" step. Arrises due to conflict with existing git on the system

```
stow --dir=dotfiles --target ~ -v git
WARNING! stowing git would cause conflicts:
  * existing target is neither a link nor a directory: .gitconfig
All operations aborted.
make: *** [dotfiles-setup] Error 1
```

MacOS High Sierra 10.13.4 with git version 2.16.2 (homebrew)

--- a/Makefile
+++ b/Makefile
@@ -39,12 +39,13 @@ dvim-setup:
 	docker build -t mshytikov/dvim:$$UID --build-arg UID=$$UID ./docker/dvim/local
 
 dotfiles-setup:
-	rm -rf  ~/.bashrc
 	mkdir -p ~/bin
 	stow --target ~/bin -v  bin
-	stow --dir=dotfiles --target ~ -v bash
-	stow --dir=dotfiles --target ~ -v git
-	stow --dir=dotfiles --target ~ -v tmux
+	stow --adopt --dir=dotfiles --target ~ -v bash
+	stow --adopt --dir=dotfiles --target ~ -v git
+	stow --adopt --dir=dotfiles --target ~ -v tmux
+	# Make backup to git stash for original dotfiles files
+	git diff --quiet dotfiles || git stash push -- dotfiles
 
 macbook-pro-setup:
 	brew ls --versions tmux            || brew install tmux

--- a/README.md
+++ b/README.md
@@ -7,23 +7,27 @@ Some intro slides: [here](https://www.slideshare.net/MaxShytikov/cloudbook-pro)
 
 ### Setup steps.
 Please review the source code before.
+
 The setup will replace dotfiles like `.bashrc` `.gitconfig` etc.
+The backup of original dotfiles files stored in `git stash`.
 
 #### Local machine
 
+Make sure you have Docker installed.
+
 Checkout project
 ```
-git clone https://github.com/mshytikov/cloudbook-pro.git && cd cloudbook-pro
+git clone https://github.com/mshytikov/cloudbook-pro.git
 ```
 
 Setup instance
 ```
-  make instance-setup
+ cd ~/cloudbook-pro && make instance-setup
 ```
 
 Restart terminal
 ```
-  make secrets-setup
+ cd ~/cloudbook-pro && make secrets-setup
 ```
 
 Activate gcloud
@@ -32,13 +36,16 @@ dgcp gcloud auth login
 ```
 
 Update ssh keys and ips in
-`cloudbook-pro/gcloud/infrastructure/cloudbook-pro.yml`
+`~/cloudbook-pro/gcloud/infrastructure/cloudbook-pro.yml`
+
+Update author name in
+`~/cloudbook-pro/dotfiles/git/.gitconfig`
 
 
 #### Create cloudbook-pro
 Create `cloudbook-pro`:
 ```
-make cloudbook-pro-create
+cd ~/cloudbook-pro && make cloudbook-pro-create
 ```
 
 ssh to the instance
@@ -46,19 +53,27 @@ ssh to the instance
 cloudbook-pro ssh
 ```
 
+install make
+```
+sudo apt-get update && sudo apt-get install make
+```
+
 checkout project
 ```
-git clone https://github.com/mshytikov/cloudbook-pro.git && cd cloudbook-pro
+git clone https://github.com/mshytikov/cloudbook-pro.git
 ```
 
 Setup instance
 ```
-  make instance-setup
+  cd ~/cloudbook-pro && make instance-setup
 ```
 
-Restart terminal
+It will fail after docker installation due to permission issue.
+Just restart terminal
 ```
-  make secrets-setup
+  cloudbook ssh
+  cd ~/cloudbook-pro && make instance-setup
+  cd ~/cloudbook-pro && make secrets-setup
 ```
 
 ### Delete cloudbook-pro

--- a/README.md
+++ b/README.md
@@ -3,8 +3,11 @@
 This is my repository of personal configuration for development env.
 I named it 'CloudBook-Pro'.  Feel free to look around.
 
+Some intro slides: [here](https://www.slideshare.net/MaxShytikov/cloudbook-pro)
+
 ### Setup steps.
 Please review the source code before.
+The setup will replace dotfiles like `.bashrc` `.gitconfig` etc.
 
 #### Local machine
 

--- a/cloudbook-pro/gcloud/infrastructure/components/3_disk.jinja
+++ b/cloudbook-pro/gcloud/infrastructure/components/3_disk.jinja
@@ -8,5 +8,5 @@ resources:
     zone: {{ zone }}
     description: CloudBook Pro
     sizeGb: {{ disk_size }}
-    sourceImage: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts
+    sourceImage: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-1804-lts
     type: zones/{{ zone }}/diskTypes/pd-ssd

--- a/dotfiles/bash/.bash_logout
+++ b/dotfiles/bash/.bash_logout
@@ -1,0 +1,7 @@
+# ~/.bash_logout: executed by bash(1) when login shell exits.
+
+# when leaving the console clear the screen to increase privacy
+
+if [ "$SHLVL" = 1 ]; then
+    [ -x /usr/bin/clear_console ] && /usr/bin/clear_console -q
+fi

